WireUI/Sources/WireIndividualToTeamMigrationUI/Models/TeamPlanFeature.swift

```diff
@@ -27,3 +27,39 @@ public struct TeamPlanFeature: Hashable, Identifiable, Sendable {
         self.description = description
     }
 }
+
+extension [TeamPlanFeature] {
+    static var features: [TeamPlanFeature] {
+        [
+            .init(
+                id: "console",
+                description: .localizedMarkdown(
+                    key: "individualToTeam.planSelection.feature.adminConsole",
+                    bundle: .module
+                )
+            ),
+            .init(
+                id: "collaboration",
+                description: .localizedMarkdown(
+                    key: "individualToTeam.planSelection.feature.collaboration",
+                    bundle: .module
+                )
+            ),
+            .init(
+                id: "meetings",
+                description: .localizedMarkdown(key: "individualToTeam.planSelection.feature.meetings", bundle: .module)
+            ),
+            .init(
+                id: "status",
+                description: .localizedMarkdown(key: "individualToTeam.planSelection.feature.status", bundle: .module)
+            ),
+            .init(
+                id: "enterprise",
+                description: .localizedMarkdown(
+                    key: "individualToTeam.planSelection.feature.enterprise",
+                    bundle: .module
+                )
+            )
+        ]
+    }
+}
```

Comment:
Not a request for a change since I understand this might be legacy code here but the `TeamPlanFeature` struct seems to be a good candidate for an enum, something like this:

```
enum TeamPlanFeature: String {
    case console
    case collaboration
    {...}
    
    var description: AttributedString {
        switch self {
        case .console:
            return .localizedMarkdown(key: "individualToTeam.planSelection.feature.adminConsole", bundle: .module)
        case .collaboration:
            return .localizedMarkdown(key: "individualToTeam.planSelection.feature.collaboration", bundle: .module)
        {...}
        }
    }
}
```

Reply:
Yes and no. It's not logic. It's content. We shouldn't even need to know the list of features.